app/database_etl/test_evaluation_handler/test_eval_link_handler.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+
+TEST_EVAL_CSV_LOCATION = 'test_evaluation_handler/test_evaluations.csv'
+
+
+def get_isotype_list(isotype_str):
+    if pd.isna(isotype_str): return []
+    if isotype_str == "Total Ig": return ["IgM", "IgG", "IgA"]
+    isotypes = isotype_str.split('; ')
+    return set(isotypes)
+
+
+test_eval_df = pd.read_csv(TEST_EVAL_CSV_LOCATION)
+test_eval_df['Target'] = test_eval_df.apply(lambda x: get_isotype_list(x['Target']), axis=1)
+manufacturer_mapping = {
+    'Vircell Microbiologists': 'Vircell S.L.',
+    'SNIBE – Shenzhen New Industries Biomedical Engineering Co.': 'Snibe Co., Ltd',
+    'Zhejiang Orient Gene (Biotech Co LTD China)': 'Orient Gene Biotech',
+    'VivaCheck Biotech': 'VivaChek',
+    'Roche Diagnostics': 'Roche'
+}
+
+
```

Comment:
Overall: do you think it would be worthwhile to abstract the components of this loop[ out a little bit, so folks can see what's going on more clearly? To make clear that you're applying a series of prioritization criteria. The estimate prioritization code provides an example of this. 

Some pseudocode for this could be something like:

```
def get_eval(estimate):
  matches = get_matching_test_manufacturer_name(estimate)
  if not matches: return [None, None]

  for prioritization_function in [prioritize_specimen_type, prioritize_reference_specimen, prioritize_isotype]:
    new_matches = prioritization_function(matches)
    if len(new_matches) == 0: continue
    elif len(new_matches) == 1: return new_matches[0]
    else: matches = new_matches
```